src/deprecated_src/ss_fact_version1.jl

```diff
@@ -0,0 +1,130 @@
+
```

Comment:
do we need to `include` this file?

Reply:
Not at the moment. `ss_fact` is basically a duplicate of this file with minor changes. But I want to keep it so I can do a line by line comparison and see what are the changes.